.github/workflows/gradle.yml

```diff
@@ -0,0 +1,71 @@
+name: Build & Test
```

Comment:
Also, it's better to add [a status badge](https://docs.github.com/en/actions/how-tos/monitor-workflows/add-a-status-badge) to README file for clarity.